conjure-java-jackson-serialization/src/test/java/com/palantir/conjure/java/serialization/BinaryBytesDeserializationModuleTest.java

```diff
@@ -0,0 +1,124 @@
+/*
+ * (c) Copyright 2025 Palantir Technologies Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.palantir.conjure.java.serialization;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
+import com.palantir.conjure.java.lib.Bytes;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.util.Base64;
+import java.util.Map;
+import java.util.Set;
+import java.util.UUID;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+public final class BinaryBytesDeserializationModuleTest {
+
+    private ObjectMapper smileMapper;
+
+    @BeforeEach
+    void before() {
+        smileMapper = ObjectMappers.newSmileClientObjectMapper();
+        smileMapper.registerModule(new BinaryBytesDeserializationModule());
```

Comment:
Slight preference to either make this a factory method returning a smile mapper, or copy/paste it into the relevant tests.
Not a strong opinion though, It's just a little odd when a test declares fields that aren't used in every test.